account_payment_restrict_mandate/tests/test_account_payment_restrict_mandate.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.tests.common import TransactionCase
+
+
+class TestAccountPaymentRestrictMandate(TransactionCase):
+    def test_account_payment_restrict_mandate(self):
+        mandate = self.env['account.banking.mandate'].create({
+            'partner_bank_id': self.env.ref('base.res_partner_2').bank_ids
+            .filtered(lambda x: x.state == 'iban').id,
+            'max_amount_per_date': 600,
+            'date_start': '2016-01-01 00:00:00',
```

Comment:
I used fixed dates on purpose to exclude any date dependent issues. Not that the code that looks for the end date prefers `date_scheduled` over `datetime.now()`: https://github.com/OCA/bank-payment/pull/281/files/9cad92470583a29415463b676b5a6ae359d27e12#diff-e1403f56fc522add18775d3d1b68ca1cR23
Or where do you expect it to break?